docker/CMSRucioClient/scripts/syncRolesToAttributesLocalUsers.py

```diff
@@ -44,112 +93,162 @@ def set_rse_manager(client, rse_name, site_managers, alt_rse=None):
     rule_approvers = ','.join(site_managers[alt_rse])
     print("Setting managers for %s to %s" % (rse_name, rule_approvers))
 
-    client.add_rse_attribute(rse=rse_name, key='rule_approvers', value=rule_approvers)
+    client.add_rse_attribute(
```

Comment:
There is no need to split this into two lines. It's just confusing.

Reply:
Yes, I find it confusing as well :sweat_smile: . My editor does that in accordance with the PEP8 style guide. I will manually revert it back to single lines, for function calls at least.

Reply:
Can you adjust where it does this? A line length of 132 is fine and your editor may be defaulting to 80. Which editor are you using?

Reply:
https://stackoverflow.com/questions/47406741/disable-auto-wrap-long-line-in-visual-studio-code